rpcs3/Emu/CPU/CPUThread.cpp

```diff
@@ -28,6 +28,7 @@ void fmt_class_string<cpu_flag>::format(std::string& out, u64 arg)
 		case cpu_flag::dbg_pause: return "PAUSE";
 		case cpu_flag::dbg_step: return "STEP";
 		case cpu_flag::__bitset_enum_max: break;
+		default: break;
```

Comment:
That's not supposed to be silenced like this... Need to add missing entries

Reply:
That's a little outside my scope. I can have it throw an error or print a warning instead, if you prefer. Would that work?

Reply:
Neither. Each enum should return a name. What name doesn't matter that much.